File: src/main/java/no/novari/flyt/egrunnerverv/gateway/instance/model/EgrunnervervJournalpostReceiver.java
package no.novari.flyt.egrunnerverv.gateway.instance.model;

import jakarta.validation.constraints.NotNull;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.extern.jackson.Jacksonized;

@Getter
@Jacksonized
@EqualsAndHashCode
@Builder
public class EgrunnervervJournalpostReceiver {
    @NotNull
    private final String navn;
    @NotNull
    private final String organisasjonsnummer;
    @NotNull
    private final String epost;
    @NotNull
    private final String telefon;
    @NotNull
    private final String postadresse;
    @NotNull
    private final String postnummer;
    @NotNull
    private final String poststed;
}

File: src/main/kotlin/no/novari/flyt/egrunnerverv/gateway/instance/model/EgrunnervervJournalpostReceiver.kt
package no.novari.flyt.egrunnerverv.gateway.instance.model

data class EgrunnervervJournalpostReceiver(
    val navn: String,
    val organisasjonsnummer: String,
    val epost: String,
    val telefon: String,
    val postadresse: String,
    val postnummer: String,
    val poststed: String,
)
